src/types/__tests__/profile.test.ts

```diff
@@ -127,4 +127,15 @@ describe("profile type", () => {
       expect(userDataKO._tag).toBe("Left");
     }
   });
+  it("should get an app Proxy profile user profile with accepted_tos_version undefined ", async () => {
```

Comment:
separate this test with the previous with an empty line - fix this text

Reply:
ok

Reply:
is the text fixed? it doesn't look correct "an app Proxy profile user profile"